content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md

```diff
@@ -99,8 +105,9 @@ Out of the box, Grafana dashboard comes pre-configured with a Prometheus data-so
 
 **Memory Resources**
 
-To run Prometheus and Grafana, you need at least 2GB to 3GB of Memory.
-If you use Minikube and you want Prometheus and Grafana running in it, you need to be sure to allocate enough resources. The instructions above point to `minikube start --cpus=4 --memory=8192`, but to account for these two components, you need at least 10GB or more of memory.
+To run Prometheus and Grafana, you need at least an additional 2GB to 3GB of Memory.
```

Comment:
Can we reverse the order here? It'd be good to order the "default" CPU/memory and then the additional how many ever gig's for the prom stack.

Reply:
updated